diagnostic.md

```diff
@@ -73,14 +88,30 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl --include --request POST "http://www.example.com/donuts" \
+  --header "Content-Type: application/json" \
+  --data "{
+    \"donut\" : {
+      \"name\" : \"French Cruller\",
+      \"price\" : \"0.99\"
+    }
+  }"
 ```
 
 Write an AJAX request to create a single donut on the server using JSON. Please
 do not use `data = getFormFields(form)` instead write out the data object.
 
 ```js
-let createDonut = /* your answer here */;
+const createDonut = function (name, price){
+  return $.ajax({
+    url: "http://www.example.com/donuts",
+    method: 'POST',
+    data: {
+      name: this.name,
+      price: this.price
+    }
+  }
+});
 ```
```

Comment:
1/2.  I can see what you want to do but you don't need to use "this" here (in this context "this" means something different than you think. Just use name:name and price:price.